### DISCUSSION TITLE
What Should Be Enterprise Level Architecture For MF?

### DISCUSSION THREAD

**Discussion by mjRaza** [module-federation/module-federation-examples#650] [Q&A]
I am making custom controls suppose i am creating a inputbox/checkbox/button/form and want these controls to be used in my apps rather than using it directly from any other component library what should I do?

1) create that controls in one micro fe and expose it to other so that they can use? will this make app slower because of runtime bundling?
2) create that controls separately in each micro fe ? but it will add repetition of code but will be faster.

**Comment by ScriptedAlchemy** [comment]
Booking a Calendly session with me and pulling me into consulting or workshop with your company. I need to understand your system and h withhold about 60% of the knowledge about MF. Users only know about less than half of what it can do
